### DISCUSSION TITLE
ROS2 Create 3 Example Projects and Humble

### DISCUSSION THREAD

**Discussion by swilloughbypme** [iRobotEducation/create3_docs#295] [Questions] [answered]
I have ROS2 Humble running on a VM, and I updated my Create 3 to the H.0.0 Humble firmware, etc. I can run terminal commands on the VM that undock/dock, change the light ring, etc. 

I'm dying to use the Create 3 examples, but I don't see a Humble branch in the create3_examples repository. I'm about 99% sure that there's no point in trying to install the Galactic version of the examples. Is that correct? If so, any idea when the Humble version will be available?

Thanks,
Steve

**Comment by alsora** [comment, marked as answer]
Hi @retupmoc1,
actually, using ROS 2 Galactic code with ROS 2 Humble is usually quite straight forward.

In the case of the Create 3 examples, I simply had to make some minor changes to reflect the new names of some Create 3 messages and actions.
I just pushed a branch named `humble`: https://github.com/iRobotEducation/create3_examples/tree/humble
it builds correctly, but it isn't fully tested, although I don't expect relevant issues.

Please have a try and let us know if you find any problems!

**Reply by swilloughbypme** [reply]
Hi @alsora,

I cloned the humble branch, and everything built successfully. The only glitch was that there was no local_setup.sh file in the workspace install folder. Copied one from another workspace, and all seems to be working as expected.

Thanks a million!
Steve
